grails-app/taglib/org/pih/warehouse/AuthTagLib.groovy

```diff
@@ -43,6 +43,18 @@ class AuthTagLib {
         }
     }
 
+    def isNonInventoryManagedAndCanSubmitRequest = {attrs, body ->
```

Comment:
Please do one more rename here from `isNonInventoryManagedAndCanSubmitRequest` to `hideIfIsNonInventoryManagedAndCanSubmitRequest`